packages/ui/components/Table/index.tsx

```diff
@@ -13,6 +14,7 @@ const TableNamespace = Object.assign(Table, {
   Row: TableRow,
   Cell: TableCell,
   CellHeader: TableCellHeader,
+  Footer: TableFooter,
```

Comment:
Is this a relevant change for this PR?

Reply:
No but because of the test coverage requirements I needed to add additional tests and this component had poor coverage